src/rpc/rawtransaction.cpp

```diff
@@ -377,7 +377,7 @@ UniValue createrawtransaction(const JSONRPCRequest& request)
             "      \"data\": \"hex\"      (string, required) The key is \"data\", the value is hex encoded data\n"
             "      ,...\n"
             "    }\n"
-            "3. locktime                  (numeric, optional, default=0) Raw locktime. Non-0 value also locktime-activates inputs\n"
+            "4. opt_into_rbf              (boolean, optional, default=false) Allow this transaction to be replaced by a transaction with heigher fees\n"
```

Comment:
When sequences aren't provided, this value does decide them, and defaults to false. I'll combine the two...